tasks/legacy_update_profile.py

```diff
@@ -0,0 +1,214 @@
+import os
+import tempfile
+
+from pathlib import Path
+from cumulusci.salesforce_api.metadata import ApiRetrieveUnpackaged
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.core.utils import process_bool_arg
+from cumulusci.tasks.salesforce import Deploy
+from cumulusci.utils import CUMULUSCI_PATH
+from cumulusci.utils import elementtree_parse_file
+
+
+class UpdateProfile(Deploy):
+    name = "UpdateProfile"
+
+    task_options = {
+        "package_xml": {
+            "description": "Override the default package.xml file for retrieving the Admin.profile and all objects and classes that need to be included by providing a path to your custom package.xml"
+        },
+        "record_types": {
+            "description": "A list of dictionaries containing the required key `record_type` with a value specifying the record type in format <object>.<developer_name>.  Record type names can use the token strings {managed} and {namespaced_org} for namespace prefix injection as needed.  By default, all listed record types will be set to visible and not default.  Use the additional keys `visible`, `default`, and `person_account_default` set to true/false to override.  NOTE: Setting record_types is only supported in cumulusci.yml, command line override is not supported."
+        },
+        "managed": {
+            "description": "If True, uses the namespace prefix where appropriate.  Use if running against an org with the managed package installed.  Defaults to False"
+        },
+        "namespaced_org": {
+            "description": "If True, attempts to prefix all unmanaged metadata references with the namespace prefix for deployment to the packaging org or a namespaced scratch org.  Defaults to False"
+        },
+        "profile_name": {
+            "description": "Name of the Profile to target for updates.",
+            "default": "Admin",
+        },
+    }
+
+    namespaces = {"sf": "http://soap.sforce.com/2006/04/metadata"}
+
+    def _init_options(self, kwargs):
+        super(UpdateProfile, self)._init_options(kwargs)
+
+        self.options["managed"] = process_bool_arg(self.options.get("managed", False))
+
+        self.options["namespaced_org"] = process_bool_arg(
+            self.options.get("namespaced_org", False)
+        )
+
+        # For namespaced orgs, managed should always be True
+        if self.options["namespaced_org"]:
+            self.options["managed"] = True
+
+        # Set up namespace prefix strings
+        namespace_prefix = "{}__".format(
+            self.project_config.project__package__namespace
+        )
+        self.namespace_prefixes = {
+            "managed": namespace_prefix if self.options["managed"] else "",
+            "namespaced_org": namespace_prefix
```

Comment:
Same as EDA change to 
```suggestion
            "namespaced_org": namespace_prefix if self.options["namespaced_org"] else "",
```

Reply:
This is `black` autoformatting; actually couldn't commit that because `black` will reformat it this way.